scripts/update_volume_monitor_aggressiveness.py

```diff
@@ -78,7 +78,12 @@ def update_aggressiveness_retrain_and_enable(monitors):
         ]:
             continue
 
```

Comment:
shouldn't we also skip the monitors which have training in progress?

Reply:
I assume that the backend will reject that configuration.

Reply:
Also, I think it would be fine to assume this is not the common case.